.travis.yml

```
@@ -18,7 +18,10 @@ script:
   - npm run lint
   - cd ..
 after_success:
-  - if [ "$TRAVIS_BRANCH" == "develop" ] && [ "$TRAVIS_PULL_REQUEST" = false ]; then
+  - if [ "$TRAVIS_REPO_SLUG" == "codeforpoznan/pah-fm"] && \
```

This won't work, unfortunately because of single space missing.
Try this instead:
```suggestion
  - if [ "$TRAVIS_REPO_SLUG" == "codeforpoznan/pah-fm" ] && \
```